.pre-commit-config.yaml

```diff
@@ -1,19 +1,19 @@
-repos:
-  - repo: https://github.com/ambv/black
-    rev: 22.3.0
-    hooks:
-    - id: black
-  - repo: https://github.com/nbQA-dev/nbQA
-    rev: 1.2.3
-    hooks:
-    - id: nbqa-black
-    - id: nbqa-isort 
-    - id: nbqa-flake8
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
-    hooks:
-    - id: isort
-  - repo: https://github.com/pycqa/flake8
-    rev: 4.0.1
-    hooks:
-    - id: flake8
+repos:
+  - repo: https://github.com/ambv/black
+    rev: 22.3.0
+    hooks:
+    - id: black
+  - repo: https://github.com/nbQA-dev/nbQA
+    rev: 1.2.3
+    hooks:
+    - id: nbqa-black
+    - id: nbqa-isort 
+    - id: nbqa-flake8
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+    - id: isort
+  - repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
+    hooks:
+    - id: flake8

```

Comment:
What change is actually being made to this file? If this is just some git oddity and this file was meant to be left untouched, then can it be removed from this commit.

Reply:
I don't know why git picked it up. I removed it from the commit.